app/controllers/mfa_confirmation_controller.rb

```diff
@@ -47,4 +48,8 @@ def after_skip_path
   def backup_code_confirmation_needed?
     !MfaPolicy.new(current_user).multiple_factors_enabled? && user_backup_codes_configured?
   end
+
+  def webauthn_platform_set_up_successful?
+    mfa_context.webauthn_platform_configurations.present?
+  end
```

Comment:
Just to check my understanding, this screen is only shown after the user sets up their first MFA method, right?

Reply:
Correct. We do not return to the confirmation screen if a user sets up F/T unlock on an existing account, that would just return the user to the account page. 

I did ask the team whether to show the updated banner after setup on an existing account. They liked the idea, so I changed the message wholesale.